code/modules/spells/spell_types/wizard.dm

```diff
@@ -354,4 +354,19 @@
 		var/mob/living/carbon/C = user
 		playsound(C.loc, 'sound/voice/hiss5.ogg', 80, 1, 1)
 		C.spin(6,1)
+	for(var/atom/movable/AM in thrownatoms)
+		if(distfromcaster == 0)

```

Comment:
this is for people exactly on the Xeno, is this really what you wanted? as this will only happen with people who are already knocked down (and with the Xeno stood on them)